### PR TITLE
Cross-referenced OSU sites (fully and uniformly)

### PR DESCRIPTION
**Pull request by MarcelWaldvogel**
This is part of a series of splits resulting from #19536 as asked

### CODE REVIEW

File: src/chrome/content/rules/Oregon-State-University.xml [hunk 1]
```diff
@@ -3,8 +3,9 @@
 Disabled by https-everywhere-checker because:
 Fetch error: http://osulibrary.oregonstate.edu/ => https://osulibrary.oregonstate.edu/: (60, 'SSL certificate problem: unable to get local issuer certificate')
 
-	Other Oregon State University rulesets:
+	For other Oregon State University coverage, see
 
+		- Oregon-State-University-mismatched.xml
```
Comment (pipboy96) [inline src/chrome/content/rules/Oregon-State-University.xml:8]:
```suggestion
```

File: src/chrome/content/rules/osufoundation.org.xml [hunk 2]
```diff
@@ -1,10 +1,14 @@
 <!--
 	For other Oregon State University coverage, see
 
+		- Oregon-State-University-mismatched.xml
```
Comment (pipboy96) [inline src/chrome/content/rules/osufoundation.org.xml:4]:
```suggestion
```

File: src/chrome/content/rules/osuosl.org.xml [hunk 3]
```diff
@@ -1,4 +1,11 @@
 <!--
+	For other Oregon State University coverage, see
+
+		- Oregon-State-University-mismatched.xml
```
Comment (pipboy96) [inline src/chrome/content/rules/osuosl.org.xml:4]:
```suggestion
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]

File: src/chrome/content/rules/osuosl.org.xml (hunk 3 above)
Comment (MarcelWaldvogel) [reply src/chrome/content/rules/osuosl.org.xml]: You mean, correcting the `*-mismatched*` file name references to `*-mismatches*`? Thanks for identifying them. Done and pushed.

**Pull request closed without merging**